app/views/meeting.html

```diff
@@ -102,8 +102,9 @@ <h4 ng-class="ng-cloak">{{ctrl.header}}</h4>
             <!-- LOWER START -->
             <div class="ad-flex-item-lower ng-class:[{'ad-flex-show-full': ctrl.isLowerMode(), 'ad-flex-show': ctrl.isBothMode(), 'ad-flex-hide': ctrl.isUpperMode()}]">
 
-                <db-pdf ng-if="ctrl.showEmbeddedFile(ctrl.lowerUrl)" ng-attr-uri="ctrl.lowerUrl" ng-attr-mode="ctrl.blockMode"></db-pdf>
+                <db-pdf ng-if="ctrl.isActive(ctrl.lbms.ATTACHMENTS || ctrl.isActive(ctrl.lbms.MATERIALS))" ng-attr-uri="ctrl.lowerUrl" ng-attr-mode="ctrl.blockMode"></db-pdf>
```

Comment:
Since only one directive can be active, a future improvement would be to use ngSwitch 

`
            <div ng-switch="ctrl.lbm">
                <div ng-switch-when="...">
                    <directive/>
                </div>
                <div ng-switch-when="...">
                    <directive/>
                </div>
                <div ng-switch-default>
                    <directive/>
                </div>
`